README.md

```diff
@@ -97,6 +97,18 @@ In `router.ex` of your Phoenix app, make sure your locally running Phoenix app c
     plug :put_secure_browser_headers, %{"content-security-policy" => @csp}
 ```
 
+### Dead View Mode
```

Comment:
1. It cannot be merged to README until we release v0.3.0
2. Currently you describe a feature in `Getting Started Section`
3. Tbh I'd move it to this section above the movie where features are listed in a short way
4. If we want to describe dead view mode in a more detailed way I think we do it inside hexdocs
5. I think we can describe each feature (by every I mean the debugging features) in hexdocs